src/core/modules/memory/memory_wrap.h

```diff
@@ -48,14 +56,55 @@ using namespace boost::python;
 class ICallingConventionWrapper: public ICallingConvention, public wrapper<ICallingConvention>
 {
 public:
-	ICallingConventionWrapper(object oArgTypes, DataType_t returnType, int iAlignment=4)
+	ICallingConventionWrapper(object oArgTypes, DataType_t returnType, int iAlignment=4, Convention_t eDefaultConv=CONV_CUSTOM)
 		:ICallingConvention(ObjectToDataTypeVector(oArgTypes), returnType, iAlignment)
 	{
+#ifdef _WIN32
+		switch (eDefaultConv)
+		{
+		case CONV_CUSTOM:
+			break;
+		case CONV_CDECL:
+			m_pCallingConvention = new x86MsCdecl(m_vecArgTypes, m_returnType, m_iAlignment);
+			break;
+		case CONV_THISCALL:
+			m_pCallingConvention = new x86MsThiscall(m_vecArgTypes, m_returnType, m_iAlignment);
+			break;
+		case CONV_STDCALL:
+			m_pCallingConvention = new x86MsStdcall(m_vecArgTypes, m_returnType, m_iAlignment);
+			break;
+		default:
+			BOOST_RAISE_EXCEPTION(PyExc_ValueError, "Unsupported calling convention.")
+		}
+#else
+		switch (eDefaultConv)
+		{
+		case CONV_CUSTOM:
+			break;
+		case CONV_CDECL:
+			m_pCallingConvention = new x86GccCdecl(m_vecArgTypes, m_returnType, m_iAlignment);
+			break;
+		case CONV_THISCALL:
+			m_pCallingConvention = new x86GccThiscall(m_vecArgTypes, m_returnType, m_iAlignment);
+			break;
+		default:
+			BOOST_RAISE_EXCEPTION(PyExc_ValueError, "Unsupported calling convention.")
+		}
+#endif
+	}
```

Comment:
We could use ``MakeDynamicHooksConvention`` for this.

Reply:
Fixed! https://github.com/Source-Python-Dev-Team/Source.Python/pull/344/commits/fb87d77359cac7e5b7b8fa32f5fca70b1e02e08f